packages/office-addin-lint/package.json

```diff
@@ -79,6 +79,12 @@
     "extends": [
       "prettier",
       "prettier/@typescript-eslint"
+    ],
+    "prettier/prettier": [
```

Comment:
It would be good to discuss the ramification of this change. Without this option, I believe it normalizes to use LF line endings. By setting "endOfLine" to "auto", how does this affect Windows and non-Windows platforms? Does it change to normalize to CRLF, or does it use whatever is there?

Also, I think this is related to VS Code settings. Is there a change that could be made in VS Code so it allows using LF line endings rather than changing this?

Reply:
I will say that for the other React-based web app that I am working on, it was changed to use "endOfLine": "auto" so this is probably a worthwhile change. I just want to probe deeper and make sure we are making an informed decision on this.

Reply:
Also, did prettier recently change the line ending behavior?

Reply:
https://prettier.io/docs/en/options.html#end-of-line

Reply:
When I look at this, I think it is good to change to "auto".